src/wavm/io.cpp

```diff
@@ -849,6 +863,19 @@ WAVM_DEFINE_INTRINSIC_FUNCTION(env,
 // Unsupported
 // -----------------------------
 
+WAVM_DEFINE_INTRINSIC_FUNCTION(env,
+                               "__small_sprintf",
```

Comment:
This is an Emscripten function but doesn't actually get called so it's not worth implementing